pkg/oc/oc.go

```diff
@@ -35,3 +35,7 @@ func LinkSecretToSA(secretname, sa, namespace string) {
 func CreateSecretWithSecretToken(secretname, namespace string) {
 	log.Printf("output: %s\n", cmd.MustSucceed("oc", "create", "secret", "generic", secretname, "--from-literal=secretToken="+os.Getenv("SECRET_TOKEN"), "-n", namespace).Stdout())
 }
+
+func DefaultTLSConfigForEventlisteners(namespace string) {
```

Comment:
imo function name should change because you are disabling the default behavior

How about this `DisableDefaultTLSConfigForEventlisteners` ?

Reply:
looks good! :heart: